### PR TITLE
adjust CSS of military history section

### PR DESCRIPTION
**Pull request by erikphansen**
## Description
- Addresses some feedback @tressaellen had here https://github.com/department-of-veterans-affairs/va.gov-team/issues/5354#issuecomment-613216679
- Addresses some other issues I found
- Cleans up markup

## Testing done
Local

## Screenshots
<details>
  <summary>Mobile</summary>
  
Screen Shot 2020-04-14 at 1 36 00 PM

</details>

<details>
  <summary>Larger screens</summary>
  
Screen Shot 2020-04-14 at 12 02 54 PM

</details>

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/personalization/profile-2/components/ProfileInfoTable.jsx [hunk 1]
```diff
@@ -1,22 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-/**
```
Comment (erikphansen) [inline src/applications/personalization/profile-2/components/ProfileInfoTable.jsx:4]: moved this to the helpers file for shared use

File: src/applications/personalization/profile-2/sass/profile-2.scss [hunk 2]
```diff
@@ -18,18 +18,6 @@ nav.profile-side-nav {
   }
 }
 
-.headerNameWrapper {
```
Comment (erikphansen) [inline src/applications/personalization/profile-2/sass/profile-2.scss:21]: woo deleted almost all the custom CSS from this file

File: src/applications/personalization/profile-2/components/ProfileHeader.jsx [hunk 3]
```diff
@@ -15,25 +15,112 @@ const ProfileHeader = ({
...
+
   return (
-    <div className="vads-u-background-color--gray-dark vads-u-color--white vads-u-margin-bottom--2 vads-u-padding-y--3 vads-u-display--flex vads-u-align-items--center vads-u-justify-content--center">
-      <div className="medium-screen:vads-u-flex-direction--row usa-grid usa-grid-full vads-u-display--flex vads-u-flex-direction--column vads-u-align-items--center vads-u-width--full">
-        <div className="medium-screen:vads-u-padding-left--8 usa-width-one-fourth">
+    <div className={[...wrapperClasses, ...wrapperClassesMedium].join(' ')}>
+      <div
+        className={[
+          // ...wrapperClasses,
+          // ...wrapperClassesMedium,
```
Comment (micahchiang) [inline src/applications/personalization/profile-2/components/ProfileHeader.jsx:86]: Should these be removed?

File: src/applications/personalization/profile-2/components/ProfileHeader.jsx [hunk 4]
```diff
@@ -15,25 +15,112 @@ const ProfileHeader = ({
...
+  const latestBranchClassesMedium = prefixUtilityClasses(
+    ['text-align--left'],
+    'medium',
+  );
+
   return (
-    <div className="vads-u-background-color--gray-dark vads-u-color--white vads-u-margin-bottom--2 vads-u-padding-y--3 vads-u-display--flex vads-u-align-items--center vads-u-justify-content--center">
-      <div className="medium-screen:vads-u-flex-direction--row usa-grid usa-grid-full vads-u-display--flex vads-u-flex-direction--column vads-u-align-items--center vads-u-width--full">
-        <div className="medium-screen:vads-u-padding-left--8 usa-width-one-fourth">
+    <div className={[...wrapperClasses, ...wrapperClassesMedium].join(' ')}>
```
Comment (micahchiang) [inline src/applications/personalization/profile-2/components/ProfileHeader.jsx:82]: Nice. Just looks so much cleaner 😍

**Review by micahchiang — APPROVED** [review]
lgtm, just left a comment about some commented-out classes, wasn't sure if they were meant to stay in there or not.

File: src/applications/personalization/profile-2/components/ProfileHeader.jsx (hunk 3 above)
Comment (erikphansen) [reply src/applications/personalization/profile-2/components/ProfileHeader.jsx]: yes, thanks!

**Pull request merged**